src/Domain/LeanCode.DomainModels.Generators/TypedIdFormat.cs

```diff
@@ -0,0 +1,9 @@
+namespace LeanCode.DomainModels.Generators;
+
+public enum TypedIdFormat : int
```

Comment:
Would be nice to coordinate this PR with ULIDs, so that a new project can embrace them both as a single breaking change.

Reply:
ULIDs must come later (and, IMO, I wouldn't allow ULIDs in the current form).